components/specification/publish

```diff
@@ -153,18 +153,23 @@ contains() {
 }
 
 devmarker="dev"
-root=published
-
+root=${1:-published}
+root=${root%/}
 
 # Clean up
 
-rm -rf "$root"
+rm -rf "$root/*"
```

Comment:
Maybe use a subdirectory in the git repository?  I don't think such logic belongs here.